components/channel_select/index.js

```diff
@@ -8,7 +8,7 @@ import {getMyChannels} from 'mattermost-redux/selectors/entities/channels';
 import {getCurrentUserLocale} from 'mattermost-redux/selectors/entities/i18n';
```

Comment:
Can we change the extension of this file to `.ts` as well and fix any issues associated?

Reply:
Agreed